app/routes/expence.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function() {
+   return this.store.findAll('expence');
+ },
+ setupController: function( controller, model) {
+        this.controller.set('model', model);
+        this.controller.set('person', this.store.findAll('person'));
```

Comment:
All the promises should be resolved in `model` hook or `beforeModel` hook and not in `setupController`.
By default, setupController will set the model property of controller to the value returned from model hook of the route.
Instead of requesting the model `person` in setupController,you should  request the  two models  by using `Ember.RSVP.hash` .For ex in your model hook you can just do this 
```
   return Ember.RSVP.hash({
        expence: this.store.find('expence'),
        person: this.store.find('person')
    });
```
And in setupController you can just write 
```
controller.set('expence',model.expence)
controller.set('person',model.person)
```